File: Iniciante/C/beecrowd1010.c
#include <stdio.h>

int main(void){

    int peca1, qtdPeca1, peca2, qtdPeca2;
    float valor1, valor2, total;

    scanf("%d%d%f",&peca1,&qtdPeca1,&valor1);
    scanf("%d%d%f",&peca2,&qtdPeca2,&valor2);

    total = qtdPeca1 * valor1 + qtdPeca2 * valor2;
    printf("VALOR A PAGAR = R$ %.2f\n", total); 

    return 0;
}

File: Iniciante/python/beecrowd1010.py
peca1 = input().split()
codigo1 = int(peca1[0])
qtdPeca1 = int(peca1[1])
valorPeca1 = float(peca1[2])

peca2 = input().split()
codigo2 = int(peca2[0])
qtdPeca2 = int(peca2[1])
valorPeca2 = float(peca2[2])

print(f'VALOR A PAGAR: R$ {(qtdPeca1*valorPeca1)+(qtdPeca2*valorPeca2):.2f}')